### ISSUE TITLE
Show meta info in the completions menu

### ISSUE THREAD

**Issue by darikg** [dbcli/pgcli#200]
Spinning this off from a derailment in #193 

Python prompt toolkit has supporting for showing extra information in the completions menu, as seen in this screenshot:

![](https://raw.githubusercontent.com/jonathanslenders/ptpython/master/docs/images/file-completion.png)

This seems to be as easy as specifying the `display_meta` parameter when constructing each `Completion` object.

We could make use of this in pgcli.

  - Indicate whether each suggestion is a schema, table, view, function, alias, column, or keyword
  - Show the quick help text for all the \special commands

**Comment by amjith** [comment]
I'm fairly certain we need to upgrade prompt_toolkit to the latest version before we can utilize this feature. So this has a hard dependency on #168.

**Comment by amjith** [comment]
Since prompt_toolkit is now upgraded, it'll be nice to add this feature. @darikg Are you interested in taking a stab?

**Comment by amjith** [comment]
This is now merged into master. Thanks @darikg for the PR.
